### PR TITLE
update resources.md

### PR DESCRIPTION
**Pull request by JV0812**
Added meta-data tag table and minor updates related to AWSO.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/deploy-use-aws-observability/resources.md [hunk 1]
```diff
@@ -195,12 +191,65 @@ If you are using an existing bucket to collect AWS ELB logs, the Amazon S3 bucke
 
 ## Resources created in Sumo Logic
 
+### Meta-data Tags
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/resources.md:194]: no dash please

```suggestion
### Metadata Tags
```

File: docs/observability/aws/deploy-use-aws-observability/resources.md [hunk 2]
```diff
@@ -195,12 +191,65 @@ If you are using an existing bucket to collect AWS ELB logs, the Amazon S3 bucke
...
+
+The Meta-data tags applied on Sumo Logic Sources.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/resources.md:196]:
```suggestion
The metadata tags applied on Sumo Logic Sources.
```

**Review by kimsauce — DISMISSED** [review]
minor fixes

File: docs/observability/aws/deploy-use-aws-observability/resources.md [hunk 3]
```diff
@@ -135,7 +131,7 @@ IAM Role
    </td>
   </tr>
   <tr>
-   <td>AWS Lambda CloudWatch logs (Lambda Log Forwarder)
+   <td>Lambda Log Forwarder ( AWS CloudWatch logs)
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/resources.md:134]:
```suggestion
   <td>Lambda Log Forwarder (AWS CloudWatch logs)
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
grammar

**Review by JainM6 — APPROVED** [review]
LTGM

**Review by JainM6 — APPROVED** [review]
LTGM

**Pull request merged**